core/Site.php

```diff
@@ -82,6 +82,13 @@ public function __construct($idsite)
         self::setSiteFromArray($this->id, $site);
 
         $this->site = $site;
+
+        // for serialized format to be predictable across php/mysql/pdo/mysqli versions, make sure the int props stay ints
+        foreach ($this->site as $key => $value) {
+            if (is_numeric($value)) {
```

Comment:
what about floats etc? It'd be actually better to really specify the properties that should be converted (like we do in other plugins) and not something generic

Reply:
changed